src/main/java/tds/exam/results/configuration/ExamResultsTransmitterServiceProperties.java

```diff
@@ -13,7 +13,10 @@
     private String tisCallbackUrl = "";
     private boolean validateTrtXml = false;
     private boolean sendToTis = false;
-    private boolean retryOnError = true;
+    private long retryInitialInterval = 1000;
+    private double retryIntervalMultiplier = 2;
+    private long retryMaxInterval = 5000;
+    private int retryAmount = 3;
```

Comment:
In that case you might want to randomize the retry time. Here's a link to a good read on this: http://dev.venntro.com/2014/07/back-off-and-retry-with-rabbitmq/.